# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/firefox/hotfix-icu4c.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- firefox-144.0.2/intl/lwbrk/LineBreaker.cpp.vanilla	2025-11-10 11:34:31.962602793 +0100
+++ firefox-144.0.2/intl/lwbrk/LineBreaker.cpp	2025-11-10 11:45:36.792035104 +0100
@@ -454,6 +454,7 @@
       /* AKSARA_START = 45,                 [AS] */ CLASS_CHARACTER,
       /* VIRAMA_FINAL = 46,                 [VF] */ CLASS_CHARACTER,
       /* VIRAMA = 47,                       [VI] */ CLASS_CHARACTER,
+      /* U_LB_UNAMBIGUOUS_HYPHEN = 48,      [HH] */ CLASS_CHARACTER,
   };
 
   static_assert(U_LB_COUNT == std::size(sUnicodeLineBreakToClass),
